.buildkite/pipelines/performance/nightly.yml

```diff
@@ -19,11 +19,19 @@ steps:
       queue: c2-16
     key: build
 
-  - label: ":muscle: Performance Tests"
-    command: .buildkite/scripts/steps/functional/performance.sh
+  - label: ":muscle: Performance Tests with FTR config"
+    command: .buildkite/scripts/steps/functional/performance_ftr.sh
     agents:
-      queue: ci-group-6
+      queue: c2-60
```

Comment:
The agent type and queue has set in this pr -> https://github.com/elastic/kibana-buildkite/pull/9